istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -233,6 +233,38 @@ func setupEnvoyLogConfig(kubeClient kube.CLIClient, param, podName, podNamespace
 	return string(result), nil
 }
 
+func printStatus(c *cobra.Command, kubeClient kube.CLIClient, statsType, podName, podNamespace string) error {
+	var stats string
+	var err error
+	if statsType == "" || statsType == "server" {
```

Comment:
We could probably make these constants (`"server", "cluster", "clusters"`) like how we handle `jsonOutput` and `yamlOutput` in this file.

Reply:
updated, thanks